package.json

```diff
@@ -42,10 +42,14 @@
     }
   ],
   "dependencies": {
-    "@githubnext/utils": "^0.11.0",
+    "@codesandbox/sandpack-react": "^0.13.13",
+    "@githubnext/utils": "^0.13.1",
     "@loadable/component": "^5.15.0",
     "@octokit/rest": "^18.12.0",
+    "@types/lodash.uniqueid": "^4.0.6",
+    "codesandbox": "^2.2.3",
```

Comment:
Are you sure we need this package? IIRC I removed this because it brings along with it a bunch of security vulnerabilities

Reply:
ah! I don't think we need it, good call. Although I'm still seeing it in the prototype - I'll remove https://github.com/githubnext/composable-github/blob/707cfdd5783e8a0b5bf72a144f5d52aa56b6cac8/package.json#L23

Reply:
Awesome, good find!